distribution/docs/src/main/resources/_contents/_core-concepts/metadata-intro-contents.adoc

```diff
@@ -0,0 +1,56 @@
+
+== Understanding Metadata and Metacards
+
+Metadata is information about a resource, organized into a schema to make it possible to search against.
+The ${ddf-catalog} stores this metadata and allows access to it.
+Metacards are single instances of metadata, representing a single record, in the Metadata Catalog (MDC).
```

Comment:
I agree. I've found, overall, we're probably close to 50/50 on `resource` v `product` and seem to use them interchangeably. I'd be for moving 100% to one or the other (leaning toward `resource`) unless there's a subtle difference that I can define here in the Intro somewhere.